tests/publiccloud/ipa.pm

```diff
@@ -48,6 +48,8 @@ sub run {
             $instance->upload_log('/var/log/cloudregister');
             last;
         }
+        $instance->run_ssh_command(cmd => 'rpm -qa > /tmp/rpm_qa.txt');
```

Comment:
rpm will work too , but why not use zypper ? 
in this regard interesting output from my laptop :
```
06:09-e113#[0] zypper se --installed-only | wc -l
6172
06:09-e113#[0] rpm -qa | wc -l
6013
```

Reply:
rpm -qa shows more information, specially the name&version of each package. With zypper it would be something like `zypper se --installed-only -s` but it also prints some other info which is not useful, like column Type or Arch. I am fine with both options.

Reply:
same from my side actually :) just wanted to clarify if it was "random" choice or has some background behind :) look like second , so LGTM :)